### PR TITLE
FIX 7342 bug, add 1004, implement PCW Flow Sensors

### PR DESCRIPTION
**Pull request by nrwslac**
During an initial scan of the EtherCat IO the scan revealed differences in found hardware vs configured hardware. Specifically IM1K0-EL7342 and IM2K0-EL7342. During the scan, each of these terminal configurations was deleted and then re-added from the found hardware. Furthermore, 2 EL9011s at the end of the configuration were not found in the scan and deleted. The new configuration was activated, terminals renamed, relinked, and CoE startups replaced.

Next, an EL1004 terminal was added to the IM1K0 Din rail: EBD-PCW-FSW-EL1004.

This EL1004 will support:
EBD-FSW2-K0-PC1
EBD-FSW4-K0-IM1
EBD-FSW5-K0-SL1

### CODE REVIEW

**Comment by ZLLentz** [comment]
> 2 EL9011s at the end of the configuration were not found in the scan and deleted

There should be no functional difference to including this in the IO tree vs not including them since they aren't even electronic devices. If it makes the scan confusing it's good to keep them out.

I'm not entirely sure why including one of these is even an option on the software side.

https://www.beckhoff.com/en-us/products/i-o/ethercat-terminals/el9xxx-system/el9011.html

File: plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 1 (EK1122)/X2 IM1K0-XTES (EK1100)/IM1K0-EL7342.xti [hunk 1]
```diff
@@ -71,11 +80,8 @@
 			<Fmmu>0000000000000000001200010100000002000000000000000000000000000000</Fmmu>
 			<Fmmu>00000000000000000d0800010100000003000000000000000000000000000000</Fmmu>
 			<BootStrapData>0010f400f410f400</BootStrapData>
-			<SuName>IM1K0</SuName>
```
Comment (ZLLentz) [inline plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 1 (EK1122)/X2 IM1K0-XTES (EK1100)/IM1K0-EL7342.xti:74]: Here's a small problem: the sync group was lost. Everything on the IM1K0 rail needs to be in the IM1K0 sync group, including the terminal you added.

File: plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 2 (EK1122)/X2 SXR EC1 (EK1100)/EC1 E1 (EK1122)/X1 IM2K0-XTES (EK1100)/IM2K0-EL7342.xti [hunk 2]
```diff
@@ -71,11 +80,8 @@
 			<Fmmu>0000000000000000001200010100000002000000000000000000000000000000</Fmmu>
 			<Fmmu>00000000000000000d0800010100000003000000000000000000000000000000</Fmmu>
 			<BootStrapData>0010f400f410f400</BootStrapData>
-			<SuName>IM2K0</SuName>
```
Comment (ZLLentz) [inline plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 2 (EK1122)/X2 SXR EC1 (EK1100)/EC1 E1 (EK1122)/X1 IM2K0-XTES (EK1100)/IM2K0-EL7342.xti:74]: Lost sync group here too on the IM2K0 EL7342

**Comment by ZLLentz** [comment]
You can ignore the CI failure, it's unrelated to this PR and is only style. I might fix it in a follow-up PR next time I work on this PLC.

File: plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 1 (EK1122)/X2 IM1K0-XTES (EK1100)/IM1K0-EL7342.xti (hunk 1 above)
Comment (nrwslac) [reply plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 1 (EK1122)/X2 IM1K0-XTES (EK1100)/IM1K0-EL7342.xti:74]: updated sync groups

File: plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 2 (EK1122)/X2 SXR EC1 (EK1100)/EC1 E1 (EK1122)/X1 IM2K0-XTES (EK1100)/IM2K0-EL7342.xti (hunk 2 above)
Comment (nrwslac) [reply plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 2 (EK1122)/X2 SXR EC1 (EK1100)/EC1 E1 (EK1122)/X1 IM2K0-XTES (EK1100)/IM2K0-EL7342.xti:74]: update sync group

**Review by ZLLentz — APPROVED** [review]
LGTM

**Pull request merged**